<launch>
  <!-- <arg name="config" default="base" /> -->
  <!-- <include file="$(find jackal_description)/launch/description.launch">
    <arg name="config" value="front_laser" />
  </include> -->
  <!-- <param name="robot_description" command="$(find xacro)/xacro.py '$(find jackal_description)/urdf/jackal.urdf.xacro'" /> -->

  <!-- <node name="static_3d_lidar_frame_publisher" pkg="tf" type="static_transform_publisher" args="0.05 0 0 0 0 0 base_link front_laser 50"/> -->
  <!-- <include file="$(find sick_scan)/launch/sick_tim_5xx.launch">
    <arg name="frame_id" value="front_laser" />
  </include> -->
 <node name="nomad_goal_sender" pkg="nomad_as_gp" type="nomad_goal_sender" />
 <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

  <rosparam file="$(find jackal_navigation)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
  <rosparam file="$(find jackal_navigation)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />

  <rosparam file="$(find jackal_navigation)/params/odom_nav_params/global_costmap_params.yaml" command="load" />
  <rosparam file="$(find jackal_navigation)/params/odom_nav_params/local_costmap_params.yaml" command="load" />

  <rosparam file="$(find jackal_navigation)/params/base_local_planner_params.yaml" command="load" />
  <rosparam file="$(find jackal_navigation)/params/move_base_params.yaml" command="load" />

  <param name="base_global_planner" type="string" value="global_planner/NomadGlobalPlanner" />
  <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/>

  <remap from="odom" to="odometry/filtered" />
 </node>
  <!-- <node name="static_3d_lidar_frame_publisher" pkg="tf" type="static_transform_publisher" args="0 0 0.1 0 0 3.14 front_mount front_laser 100"/> -->

</launch>